unittests/test_autodetect.py

```diff
@@ -110,10 +120,17 @@ def test_autotect_with_invalid_files(invalid_topo_exec_ctx):
     assert part.devices == []
 
 
+@pytest.mark.parametrize('remote_exec_ctx', [False, True], indirect=True)
```

Comment:
When parametrizing boolean fixtures, I don't prefer using the raw true/false values because they don't convey any information in the test name. Instead of `pytest.mark.parametrize`, I would parameterize a fixture and pass this to `remote_exec_ctx`:

```python
@pytest.fixture(params=['default', 'custom_install'])
def custom_install(request):
    return request.param == 'custom_install'


@pytest.fixture
def remote_exec_ctx(make_exec_ctx, temp_topo, custom_install):
```